README.md

```diff
@@ -1,33 +1,38 @@
-<img src="https://www.mediation-sa.lu/wp-content/uploads/2021/01/team1.jpg" width="650">
+<img src="https://i.ibb.co/wg0kWGD/Whats-App-Image-2021-11-08-at-5-56-01-PM.jpg" width="650">
 
 
 # Description
 
-Welcome to our project repository! Enjoy!
+Welcome to AcadeMe! :technologist:
+For students, by students. :student:
 
 
 # Installation
 
 1. Install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/)/[VMWare](https://www.vmware.com/) hypervisor.
-2. Clone [Team1Project](https://github.com/redhat-beyond/beyond-os-06-group-1) repository.
+2. Clone [Team1Project](https://github.com/redhat-beyond/AcadeMe) repository.
```

Comment:
Please change this -> AcadeMe

Please refer the following markdown guidelines 
https://www.markdownguide.org/cheat-sheet/

Reply:
Do you mean the links tag name?

Reply:
The Team1Project -> AcadeMe

Reply:
I made the changes on github, see the latest 2 commits.
Is it possible to merge it now?